ajax/libs/instagram-lite/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "instagram-lite",
+  "version": "2.0.0",
+  "filename": "instagramLite.min.js",
+  "description": "A simple, lightweight jQuery plugin used to display a user's Instagram photos.",
+  "keywords": [
+    "instagram",
+    "photos",
+    "api",
+    "feed"
+  ],
+  "license": "MIT",
+  "author": {
+    "name": "Michael Lynch",
+    "url": "http://michaelynch.com"
+  },
```

Comment:
Use the following for the `author` field.
(Ref: https://github.com/michael-lynch/instagram-lite/blob/master/bower.json#L6)

```
"author": "Michael Lynch <mikethomaslynch@gmail.com>",
```